src/tensorlake/error.py

```diff
@@ -10,3 +10,25 @@ def __str__(self):
 class GraphStillProcessing(Exception):
     def __init__(self) -> None:
         super().__init__("graph is still processing")
+
+
+class InvocationError(Exception):
```

Comment:
The existing errors in this file are coming from Server API. Would be confusing for future readers I think.

Reply:
More like api_errors.py and runtime_errors.py would work better with a comment on what's in these files.

Reply:
Done